### DISCUSSION TITLE
Does strimzi have api's to use in the java code to manage the topic, users and acl like admin client api?

### DISCUSSION THREAD

**Discussion by rishaik05** [strimzi/strimzi-kafka-operator#10925] [General]
Does strimzi have api's to use in the java code to manage the topic, users and acl like admin client api?

If present, please share the dependency packages details to add in the pom.xml file

**Comment by scholzj** [comment]
If you want to create the KafkaUser or KafkaTopic resources from Java you can use the Stirmzi `api` module. You can find some examples of it here: https://github.com/scholzj/strimzi-api-examples

**Reply by rishaik05** [reply]
Please share some examples for KafkaTopic and KafkaUser.

Facing error - java: package io.strimzi.api.kafka.model does not exist

when i used below two classes

import io.strimzi.api.kafka.model.KafkaTopic;
import io.strimzi.api.kafka.model.KafkaTopicSpec;

**Reply by scholzj** [reply]
All the examples I have are in the repository I shared.
